web/packages/design/src/theme/themes/sharedStyles.ts

```diff
@@ -30,6 +30,9 @@ export const breakpointsPx = {
   small: 600,
   medium: 1024,
   large: 1280,
+  700: 700,
+  900: 900,
+  1200: 1200,
```

Comment:
I'll mention this on Slack, but I talked with Rishi and the design team doesn't really use any preset breakpoints – they might differ page to page. However, styled-system requires breakpoints to be defined ahead of time (I think [Chakra requires that as well](https://www.chakra-ui.com/docs/theming/customization/breakpoints)).

So instead of binding them to specific names, I just used numeric values.